packages/neuron-ui/src/services/remote/index.ts

```diff
@@ -1,3 +1,13 @@
+import {
+  ipcRenderer,
+  remote,
+  desktopCapturer,
+  shell,
+  OpenDialogOptions,
+  MenuItemConstructorOptions,
```

Comment:
`import  type { A, B } from 'example'` will can cause ESLint errors.  But `import  type { A } from 'example'` will do fine.
I think maybe that's a `@typescript-eslint/parser` bug,

Reply:
![image](https://user-images.githubusercontent.com/7202516/90843107-7b817200-e393-11ea-881a-ed61379cc351.png)
If importing multiple types under the ESLint configuration of the rendering process raises an error, I'm guessing it's an ESLint bug (which would explain why the current code import types all have only one type).

Reply:
I think that's an error from `prettier`(in most cases prettier is so stupid and always outdated) since I can import 2 types in `neuron-wallet`. I also tried to remove `prettier` from `neuron-ui` and the alert was dismissed.

For now, let's just leave it.